staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -48,6 +49,9 @@ const (
 	outgoingBufSize = 100
 )
 
+// defaultWatcherMaxLimit is used to facilitate construction tests
+var defaultWatcherMaxLimit int64 = maxLimit
```

Comment:
would it make sense to set this value to match `incomingBufSize/outgoingBufSize` ?

Reply:
okay, it makes sense, thx.